ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.ts

```diff
@@ -204,100 +269,18 @@ describe('Workload CVE overview page tests', () => {
 
             assertCveElementsArePresent();
 
-            // Switch to the "without cves" view, we should stay on the deployments tab
-            interactAndWaitForDeploymentList(() => {
-                changeObservedCveViewingMode('Images without vulnerabilities');
-            });
-
-            assertCveElementsAreNotPresent();
-
-            // Visit the Images tab
             interactAndWaitForImageList(() => {
-                selectEntityTab('Image');
+                navigateToViewViaDropdown('Images without CVEs');
             });
 
             assertCveElementsAreNotPresent();
-        });
-
-        it('should apply the correct filters when switching between "with cves" and "without cves" views', () => {
```

Comment:
This test is obsolete, and is replaced with the more comprehensive test above for validating baseline filters.